staging/src/k8s.io/client-go/examples/custom-resources/README.md

```diff
@@ -4,4 +4,4 @@
 
 For a client-go example using CustomResourceDefinitions, go to
 
-  http://github.com/kubernetes/apiextensions-apiserver/examples/client-go.
+  [k8s.io/apiextensions-apiserver/examples/client-go](https://github.com/kubernetes/apiextensions-apiserver/tree/master/examples/client-go).
```

Comment:
@thockin created https://git.k8s.io for stuff like this.
Try https://git.k8s.io/apiextentions-apiserver/examples/client-go